stubs/paramiko/paramiko/win_pageant.pyi

```diff
@@ -3,12 +3,14 @@ import sys
 
 if sys.platform == "win32":
     win32con_WM_COPYDATA: int
-    def can_talk_to_agent(): ...
+    def can_talk_to_agent() -> bool: ...
+
+    ULONG_PTR: ctypes.c_uint64 | ctypes.c_unit32
```

Comment:
```suggestion
    ULONG_PTR: ctypes.c_uint64 | ctypes.c_uint32
```

Reply:
Oops, sorry about that.

Reply:
No need to be sorry, happens to us all :)